implementations/typescript/ockam/ockam_app/src/routes/invite/[outlet_addr]/CreateServiceInvitation.svelte

```diff
@@ -0,0 +1,108 @@
+<script>
+  import { appWindow } from "@tauri-apps/plugin-window";
+  import { invoke } from "@tauri-apps/api";
+
+  export let outlet_addr;
+
+  let email;
+  let error = null;
+  let outlet;
+  let outlets;
+  let project;
+  let projects;
+  let list_outlets = invoke("plugin:invitations|list_outlets").then(
+    (received_outlets) => {
+      console.debug(received_outlets);
+      outlets = received_outlets;
+      // TODO: more defensive code
+      outlet = outlets.length > 0 ? outlets[0] : null;
+    },
+  );
+  let list_projects = invoke("plugin:projects|list_projects").then(
+    (received_projects) => {
+      console.debug(received_projects);
+      projects = received_projects;
+      // TODO: more defensive code
+      project = projects.length > 0 ? projects[0] : null;
+    },
+  );
```

Comment:
Non blocking but perhaps we should simplify the `create_service_invitation` payload and build the `CreateServiceInvitation` in the command handler like it's [done here](https://github.com/build-trust/ockam/blob/30357ee24f5ce433963997d568410addc712bcf9/implementations/rust/ockam/ockam_app/src/shared_service/tcp_outlet/create.rs#L64). Perhaps I'm missing something?

Reply:
I did wind up including this redesign and I liked the outcome much better, thank you! Currently visible in c2602a01a2140697e552804139f3fabec1da1a10...d711d187e3e2daad358267bac5a981483d5ad6cd.

Reply:
Nice! 💪